src/main/components/RequestBar.jsx

```diff
@@ -16,12 +16,33 @@ const RequestBar = (props) => {
 
   function sendFetch(e) {
     e.preventDefault();
-    const sendingObj = { method: selected, mode: 'cors' };
-    if (SourceOrDest === 'dest') sendingObj.body = JSON.stringify(requestBody);
 
-    fetch(uri, sendingObj)
-      .then(res => res.json())
-      .then(res => setData(res));
+    if (SourceOrDest === 'source') {
+      const sendingObj = { method: selected, mode: 'cors' };
+
+      fetch(uri, sendingObj)
+        .then(res => res.json())
+        .then(res => setData(res));
+    }
+    else if (SourceOrDest === 'dest') {
```

Comment:
let's move this line up one to be on the same line as the closing brace of the if statement.

Reply:
I fixed this manually, was blocking other features